From f7679a4083de3a9dba80cd719ad4904fbade349b Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@gmail.com>
Date: Wed, 25 May 2016 15:57:29 +0200
Subject: [PATCH] cmake: install oplk headers files

In order to be able to link a third party application
with openpowerlink libraries, we need to install the
oplk headers files related to openpowerlink stack.

Install all headers file from STACK_INCLUDE_DIR/oplk.

Signed-off-by: Romain Naour <romain.naour@gmail.com>
---
 stack/CMakeLists.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/stack/CMakeLists.txt b/stack/CMakeLists.txt
index 69454f2..45f131d 100644
--- a/stack/CMakeLists.txt
+++ b/stack/CMakeLists.txt
@@ -110,3 +110,8 @@ ELSEIF((CMAKE_SYSTEM_NAME STREQUAL "Generic") AND (CMAKE_SYSTEM_PROCESSOR STREQU
 ELSE()
     MESSAGE(FATAL_ERROR "Unknown Platform and processor combination ${CMAKE_SYSTEM_NAME} and ${CMAKE_SYSTEM_PROCESSOR}!!")
 ENDIF()
+
+################################################################################
+# Install oplk headers files
+################################################################################
+INSTALL(DIRECTORY ${STACK_INCLUDE_DIR}/oplk DESTINATION "include")
-- 
2.9.4

